### PR TITLE
Bump to v1.0.0

### PR DESCRIPTION
**Pull request by stephenroller**
**Patch description**
- Release of v1.0.0
- Archive NEWS.md
- Update some documentation

**Testing steps**
Careful review. Manually ran build website and inspected the new homepage.

### CODE REVIEW

File: website/templates/home.html [hunk 1]
```diff
@@ -103,7 +106,7 @@ <h4 style="color: #555; margin-bottom: 20px; font-size: 16px; line-height: 140%"
         </div>
         <div class="command" style="margin-bottom: 20px">
             <div class="label">Run this command:</div>  
-            <div class="text" style="background-color: #fff"><code>parlai display_data -t babi:task1k:1,squad -n 100</code></div>
+            <div class="text" style="background-color: #fff"><code>parlai display_data --task babi:task1k:1,squad -n 100</code></div>
```
Comment (klshuster) [inline website/templates/home.html:109]: `-n` --> `--num-examples`?

File: website/templates/home.html [hunk 2]
```diff
@@ -113,7 +116,7 @@ <h4 style="color: #555; margin-bottom: 20px; font-size: 16px; line-height: 140%"
         </div>
         <div class="command" style="margin-bottom: 20px">
             <div class="label">Run this command:</div>  
-            <div class="text" style="background-color: #fff"><code>parlai eval_model -m ir_baseline -t "#moviedd-reddit" -dt valid</code></div>
+            <div class="text" style="background-color: #fff"><code>parlai eval_model --model ir_baseline --task "#moviedd-reddit" -dt valid</code></div>
```
Comment (klshuster) [inline website/templates/home.html:119]: `--datatype`

File: website/templates/home.html [hunk 3]
```diff
@@ -123,7 +126,7 @@ <h4 style="color: #555; margin-bottom: 20px; font-size: 16px; line-height: 140%"
         </div>
         <div class="command" style="margin-bottom: 20px">
             <div class="label">Run this command:</div>  
-            <div class="text" style="background-color: #fff"><code>parlai display_model -m ir_baseline -t "#moviedd-reddit" -dt valid</code></div>
+            <div class="text" style="background-color: #fff"><code>parlai display_model --model ir_baseline --task "#moviedd-reddit" -dt valid</code></div>
```
Comment (klshuster) [inline website/templates/home.html:129]: ditto

File: website/templates/home.html [hunk 4]
```diff
@@ -133,7 +136,7 @@ <h4 style="color: #555; margin-bottom: 20px; font-size: 16px; line-height: 140%"
         </div>
         <div class="command" style="margin-bottom: 20px">
             <div class="label">Run this command:</div>  
-            <div class="text" style="background-color: #fff"><code>parlai train_model -m drqa -t squad -bs 32 -mf /tmp/model_drqa</code></div>
+            <div class="text" style="background-color: #fff"><code>parlai train_model --model drqa --task squad -bs 32 -mf /tmp/model_drqa</code></div>
```
Comment (klshuster) [inline website/templates/home.html:139]: --batchsize, --model-file

**Review by klshuster — APPROVED** [review]
🎊 💯 🥳 🎉

File: website/templates/home.html (hunk 1 above)
Comment (stephenroller) [reply website/templates/home.html:109]: I feel like that one's clear enough from the description

**Pull request merged**